packages/phone-number-privacy/common/src/utils/logger.ts

```diff
@@ -5,31 +5,40 @@ import { NextFunction, Request, Response } from 'express'
 import { WarningMessage } from '../interfaces/error-utils'
 import { fetchEnv, fetchEnvOrDefault } from './config-utils'
 
-const logLevel = fetchEnvOrDefault('LOG_LEVEL', 'info') as LogLevelString
-const logFormat = fetchEnvOrDefault('LOG_FORMAT', 'human')
-const serviceName = fetchEnv('SERVICE_NAME')
-
-let stream: any
-switch (logFormat) {
-  case 'stackdriver':
-    stream = createStream(levelFromName[logLevel])
-    break
-  case 'json':
-    stream = { stream: process.stdout, level: logLevel }
-    break
-  default:
-    stream = { level: logLevel, stream: bunyanDebugStream() }
-    break
-}
+let _rootLogger: Logger | undefined
+
+export function rootLogger(): Logger {
```

Comment:
I was encountering issues with this code trying to access environment variables and returning an error on import. This was the smallest change which allowed me to import this module without it trying to access environment variable or otherwise doing work.